### ISSUE TITLE
spectral function normalization

### ISSUE THREAD

**Issue by NickRybin** [band-unfolding/banduppy#5]
Hello, @stepan-tsirkin!

I have a small question related to the normalization of the spectral function. When we use "density"  option for plotting, the spectral function has intensity, which is not from 0 to 1. In my understanding, this is not a scaling issue (cause in principle, we can scale intensities from [0,x] to [0,1]), but it is related to the normalization of the spectral function when the delta function is presented as gaussian in the code. Am I right that if the spectral function is properly normalized it should be between 0 and 1 or I am missing something? Could you, please, clarify this point?

**Comment by stepan-tsirkin** [comment]
Hi @NickRybin ,

No, the densitydoes not have to be between 0 and 1. When a delta-function is represented by a gaussian, the integral should be fixed, not the value at maximum.

**Comment by stepan-tsirkin** [comment]
@NickRybin 

More detailed : If a band is represented as a gaussian (or any other smeared peak) g(E)  it should be 

\int g(E) dE=1 

but when doing unfolding, that gaussian is multiplied by the unfolding weight W, which is between 0 and 1. But if 2 or more bands are degenerate (within the threshold), then the weights add up, so the weight may be actually greater than 1.

Does it answer your question?

**Comment by NickRybin** [comment]
Hi, Stepan! 

Yeap, this I understand, but then I calculate degeneracy wrongly. I have Si supercell (at equilibrium, i.e. zero temperature, no distortion, so all the weights should be really either 0 or 1) and for some reason the valence band maxima has an intensity ~6, whereas I though that the VBM at gamma point is three fold degenerate and shold have intensity ~3. The same I observe for the CBM at Gamma point, where the degeneracy, as I count should be 3, but somehow the intensity is also ~6.

**Comment by stepan-tsirkin** [comment]
Hi Nikita,

Yes, if you do a spinor calculation, the deeneracy is 6. The deeneracies are easier to see in the weights, e.g. here : 

https://github.com/band-unfolding/banduppy/blob/f26af6a20a9abf2c6a704fe8719936aaa9a67a33/tutorial/reference/bandstructure_unfolded.txt#L476-L503

I see only upto 3-fold degeneracies, but that is a scalar calculation.

**Comment by NickRybin** [comment]
Yes, you are right! I completely forgot about the spin. Thanks for clarifications.
